server/filestore.go

```diff
@@ -3674,7 +3686,9 @@ func (fs *fileStore) NumPending(sseq uint64, filter string, lastPerSubject bool)
 			// We need to discount the total by subjects seen before sseq, but also add them right back in if they are >= sseq for this blk.
 			// This only should be subjects we know have the last blk in this block.
 			if mb.cacheNotLoaded() {
-				mb.loadMsgsWithLock()
+				if err = mb.loadMsgsWithLock(); err != nil {
+					return
```

Comment:
I believe this is a "naked return", could this (and others for `NumPending` and `NumPendingMulti`) be changed to `return 0, 0, err` to make it more clear this `err` is actually passed upward?